tests/integration/components/modals/-body-test.js

```diff
@@ -0,0 +1,14 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | modals/ body', function(hooks) {
```

Comment:
I don't quite care  for the formatting on `Integration | Component | modals/ body` but it is not that big of a deal.

I like the `Integration | Component | modals | body` format.